.github/workflows/e2e-detox-template.yml

```diff
@@ -66,8 +66,8 @@ env:
   ADMIN_EMAIL: ${{ secrets.MM_MOBILE_E2E_ADMIN_EMAIL }}
   ADMIN_USERNAME: ${{ secrets.MM_MOBILE_E2E_ADMIN_USERNAME }}
   ADMIN_PASSWORD: ${{ secrets.MM_MOBILE_E2E_ADMIN_PASSWORD }}
-  # BRANCH: ${{ github.head_ref || github.ref_name }} 
-  # COMMIT_HASH: ${{ github.sha }}
```

Comment:
Is this intended?

Reply:
@larkox 
In reports, it always says HEAD. I'm trying to get the branch name to populate there. I see this change is not working.

I'll try some other options.
<img width="513" alt="image" src="https://github.com/user-attachments/assets/26941e9a-1997-4dbf-8809-2137e530f5f8">